crates/ruff_formatter/src/printer/mod.rs

```diff
@@ -123,7 +123,7 @@ impl<'a> Printer<'a> {
                     }
 
                     // Print a second line break if this is an empty line
-                    if line_mode == &LineMode::Empty && !self.state.has_empty_line {
+                    if line_mode == &LineMode::Empty {
                         self.print_str("\n");
                         self.state.has_empty_line = true;
```

Comment:
Is it possible to remove the `has_empty_line` state?

Reply:
Yes! Good call.